docs/Authentication.md

```diff
@@ -56,12 +56,14 @@ extension YourAPI: TargetType, AccessTokenAuthorizable {
     case targetDoesNotNeedAuth
 
     var authorizationType: AuthorizationType {
-    case targetThatNeedsBearerAuth:
-        return .bearer
-    case targetThatNeedsBasicAuth:
-        return .basic
-    case targetDoesNotNeedAuth:
-        return .none   
+    switch self {
+        case targetThatNeedsBearerAuth:
+            return .bearer
+        case targetThatNeedsBasicAuth:
+            return .basic
+        case targetDoesNotNeedAuth:
+            return .none
```

Comment:
I also noticed that the cases aren't prefixed with a full stop 😅 
It's amazing how much I take auto-complete for granted...
If you fix this can you also indent the `switch` statement?